bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java

```diff
@@ -276,7 +285,23 @@ private Addon convertTopicItemToAddon(DiscourseTopicItem topic, List<DiscourseUs
         String type = (addonType != null) ? addonType.getId() : "";
         String contentType = getContentType(topic.categoryId, tags);
 
-        String title = topic.title;
+        String title;
+        boolean compatible = true;
+
+        int compatibilityStart = topic.title.lastIndexOf("["); // version range always starts with [
```

Comment:
To prevent false positives, it might also help to only check the last word of the title.